db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150717110438) do
+ActiveRecord::Schema.define(version: 20150817111030) do
```

Comment:
I've changed the migration to add_index rather than change_column - schema now updated